static/src/javascripts/projects/common/modules/sport/football/match-info.js

```diff
@@ -0,0 +1,29 @@
+import fetch from 'lib/fetch';
+
+/**
+ * @param {Object} match
+ * @param {string} whosCalling (url)
+ */
+
+class MatchInfo {
+    constructor(match, whosCalling) {
+        const urlBase = '/football/api/match-nav';
+        const pageSlug = encodeURIComponent(whosCalling);
+        const matchSlug = [match.date].concat(match.teams).join('/');
+        const endpoint = match.id || matchSlug;
```

Comment:
I don't think it matters here, just keep in mind that `A; B;` is operationally different from `A || B`. In the latter case, `B` is not evaluated if `A` is truthy, which could matter to avoid divisions by zero or error cases.